showcases/react-showcase/src/app.tsx

```diff
@@ -12,97 +15,116 @@ const App = () => {
-					gap: '1rem',
-					margin: '1rem 0'
-				}}>
-				<section className="db-ui-expressive">
-					<DBInput
-						description="Das ist die Beschreibung"
-						label="Startbahnhof eingeben"
-						placeholder="irgendein Text"
-						iconBefore="edit"
-						variant="error"
-						value="hello"
-					/>
+						<DBInput
+							description="Valid test"
+							label="Mit Event"
+							placeholder="irgendein Text"
+							iconBefore="edit"
+							iconAfter="heart"
+							variant="warning"
+							id="input-expr-warning"
+							required={true}
+						/>
+						<DBInput
+							label="Startdatum"
+							placeholder="irgendein Text"
+							type="datetime-local"
+							id="input-expr-date"
+						/>
+					</section>
 
-					<DBInput
-						description="Valid test"
-						label="Mit Event"
-						placeholder="irgendein Text"
-						iconBefore="edit"
-						iconAfter="heart"
-						variant="warning"
-						id="input-expr-warning"
-						required={true}
-					/>
-					<DBInput
-						label="Startdatum"
-						placeholder="irgendein Text"
-						type="datetime-local"
-						id="input-expr-date"
-					/>
-				</section>
+					<section className="db-ui-regular">
+						<DBInput
+							label="Startbahnhof eingeben"
+							placeholder="irgendein Text"
+							iconAfter="heart"
+							id="input-reg"
+							required={true}
+							minLength={5}
+						/>
+						<DBInput
+							label="Startdatum"
+							placeholder="irgendein Text"
+							type="week"
+							id="input-reg-date"
+						/>
+					</section>
 
-				<section className="db-ui-regular">
-					<DBInput
-						label="Startbahnhof eingeben"
-						placeholder="irgendein Text"
-						iconAfter="heart"
-						id="input-reg"
-						required={true}
-						minLength={5}
-					/>
-					<DBInput
-						label="Startdatum"
-						placeholder="irgendein Text"
-						type="week"
-						id="input-reg-date"
-					/>
-				</section>
+					<section className="db-ui-functional">
+						<DBInput
+							id="db-input-functional-1"
+							label="Startbahnhof eingeben"
+							placeholder="irgendein Text"
+						/>
+						<DBInput
+							id="db-input-functional-2"
+							label="Textinput eingeben disabled"
+							placeholder="irgendein Text"
+							variant="information"
+							disabled={true}
+						/>
 
-				<section className="db-ui-functional">
-					<DBInput
-						id="db-input-functional-1"
-						label="Startbahnhof eingeben"
-						placeholder="irgendein Text"
-					/>
-					<DBInput
-						id="db-input-functional-2"
-						label="Textinput eingeben disabled"
-						placeholder="irgendein Text"
-						variant="information"
-						disabled={true}
-					/>
-
-					<DBInput
-						label="Startdatum"
-						placeholder="irgendein Text"
-						type="datetime-local"
-						id="input-func-date"
-					/>
-				</section>
+						<DBInput
+							label="Startdatum"
+							placeholder="irgendein Text"
+							type="datetime-local"
+							id="input-func-date"
+						/>
+					</section>
+				</div>
 			</div>
```

Comment:
```suggestion
			</main>
```

every page should contain a `<main>` element, mainly for a11y purpose to act as a landmark.

Reply:
Had to revert it...
`<DBPage>` has a `<main>` tag already: https://github.com/db-ui/mono/blob/feat-db-brand/packages/components/src/components/page/page.lite.tsx#L40

We gonna refactor the showcases in the future. At the part you mentioned would be a native router outlet (inside the main tag) that loads another component (content-site) based on the route. So I think everything is fine here